src/components/AssetSelection/AssetSelection.tsx

```diff
@@ -18,6 +18,7 @@ type TradeAssetSelectBaseProps = {
   isLoading?: boolean
   buttonProps?: ButtonProps
   onlyConnectedChains: boolean
+  showChainDropdown?: boolean
```

Comment:
FYI, last we discussed on groomsies I think the consensus was to ditch that fully, though we'll want to capture a ticket for it, so absolutely fine for now!